### PR TITLE
k8s readme: update exposing services section

### LINKED ISSUE #1656: K8s readme: services are now registered in Consul automatically
From the [1.2.0-RC2 release notes](https://github.com/CiscoCloud/mantl/releases/tag/1.2.0-RC2):

> In addition, we are excited to introduce Kubernetes integration with Consul — now your Kubernetes services will be registered in Consul. 

The README doesn't yet relflect this change:
https://github.com/CiscoCloud/mantl/blob/eb8f05938230d9f3f211a0533f581f94a6b43b54/roles/kubernetes/README.rst
- Ansible version (`ansible --version`): 1.9.4
- Python version (`python --version`): 2.7.11
- Git commit hash or branch: master
- Cloud Environment: all
- Terraform version (`terraform version`): 0.7.0

### PR DESCRIPTION
**Pull request by siddharthist**
- [x] Installs cleanly on a fresh build of most recent master branch
- [x] Upgrades cleanly from the most recent release
- [x] Updates documentation relevant to the changes
  - include LoadBalancer/Cloud Provider Integration
  - discuss automatic Consul service registration

fixes #1656

### CODE REVIEW

File: roles/kubernetes/README.rst [hunk 1]
```diff
@@ -18,11 +18,19 @@ integrates into the workflow.)
...
-To talk to the services launched inside Kubernetes, launch them with the
-``NodePort`` service type (`more on what service types are available
-<https://aster.is/blog/2016/03/11/the-hamburger-of-kubernetes-service-types/>`_),
-then connect on the assigned port on any Kubernetes worker node. Consul service
-integration will happen in a future release.
+Exposing Services
+-----------------
+
+To talk to the services launched inside Kubernetes, you can either launch them
+with the ``NodePort`` service type (all platforms), or the ``LoadBalancer``
```
Comment (ryane) [inline roles/kubernetes/README.rst]: what about the ClusterIP service type? That should be an option too, no?

http://kubernetes.io/docs/user-guide/services/#publishing-services---service-types

File: roles/kubernetes/README.rst [hunk 2]
```diff
@@ -18,11 +18,19 @@ integrates into the workflow.)
...
+service type (see the section on "Cloud Provider Integration" below). You can
+find out more about service types on `the Asteris blog
+<https://aster.is/blog/2016/03/11/the-hamburger-of-kubernetes-service-types/>`_).
+
+Your exposed Kubernetes services will automatically be registered in Consul, but
+they currently do not have valid DNS names (see `the issue on Mustwin's fork of
+Kubernetes <https://github.com/MustWin/kubernetes/issues/7>`_ for details).
```
Comment (ryane) [inline roles/kubernetes/README.rst]: The latest version of kube2consul will generate valid dns names, see #1732

File: roles/kubernetes/README.rst (hunk 1 above)
Comment (siddharthist) [inline roles/kubernetes/README.rst]: Haven't tried, I'll try on the next cluster I build.

File: roles/kubernetes/README.rst (hunk 2 above)
Comment (siddharthist) [inline roles/kubernetes/README.rst]: Will it be in 1.3?

Comment (ryane) [inline roles/kubernetes/README.rst]: that's definitely the goal

**Pull request merged**